posthog/src/test/resources/json/decide-with-stringfied-flags.json

```diff
@@ -0,0 +1,40 @@
+{
+  "autocaptureExceptions": false,
+  "toolbarParams": {},
+  "errorsWhileComputingFlags": false,
+  "capturePerformance": true,
+  "autocapture_opt_out": false,
+  "isAuthenticated": false,
+  "supportedCompression": [
+    "gzip",
+    "gzip-js"
+  ],
+  "config": {
+    "enable_collect_everything": true
+  },
+  "featureFlagPayloads": {
+    "theString": "theString",
+    "theInteger": "123",
+    "theDouble": "123.5",
+    "theObject": "{\"key\":\"value\"}",
+    "theArray": "[1, \"2\", 3.5]",
+    "theBoolean": "true",
+    "theNull": null,
+    "theBroken": "[1, 2"
```

Comment:
@neilkakkar can you confirm that this is how the API returns the stringified version of such JSONs?

Reply:
This looks alright! 👍 

If you ever want to check yourself, just open network inspector on https://app.posthog.com/home , filter for `decide`, and see "featureFlagPayloads". You can generate a random flag with broken json and see what it looks like.